src/routes/login.js

```diff
@@ -279,7 +279,7 @@ export default ({
     }
     const ids = findUserIds.result.items.map(item => item.accountId.toString())
 
-    const getAccounts = await list(AccountModel, {}, { filter: { _id: { $in: ids } } })
```

Comment:
Why don't we only return that's absolutely necessary to send? and we would not include anything else?

Reply:
so later if we add something else that's huge, it won't cause issues again.